tests_e2e/test_end_to_end.py

```diff
@@ -0,0 +1,51 @@
+import os, requests, pytest, app
+from threading import Thread
+
+from Trello_items import boardsurl, get_trello_key, get_trello_token, build_auth_query
+from Trello_boards import create_trello_board, delete_trello_board
+from dotenv import load_dotenv, find_dotenv
+
+@pytest.fixture(scope='module')
+def test_app():
+    # Create the new board & update the board id environment variable
```

Comment:
End to end tests will currently fail when running after integration tests with the lines below:
```suggestion
    # Create the new board & update the board id environment variable
    file_path = find_dotenv('.env.test')
    load_dotenv(file_path, override=True)
```